docs/book/src/reference/versions.md

```diff
@@ -57,7 +57,7 @@ The Core Provider, Kubeadm Bootstrap Provider, and Kubeadm Control Plane Provide
 In some cases, the Management Cluster is separate from the Workload Clusters. The Kubernetes version of the Management and Workload Clusters are allowed to be different.
 
 Management Clusters and Workload Clusters can be upgraded independently and in any order, however, if you are additionally moving from
-v1alpha3 (v0.3.x) to v1beta1 (v1.x) as part of the upgrade rollout, the management cluster will need to be upgraded to at least v1.20.x,
+v1alpha3 (v0.3.x) to v1beta1 (v1.x) as part of the upgrade rollout, the management cluster will need to be upgraded to at least v1.24.x,
```

Comment:
Is this true? I would have assumed that people can still migrate from v0.3 to v1.0 and nothing changed recently about the supported min Kubernetes version of v1.0

Reply:
Also I wouldn't want to say that a CAPI upgrade from v0.3 to v1.5 is actually supported (I guess that is why the version was bumped to 1.24)

Reply:
Tried to qualify better.
But open to suggestions.

Reply:
> Also I wouldn't want to say that a CAPI upgrade from v0.3 to v1.5 is actually supported (I guess that is why the version was bumped to 1.24)

I'm not sure it works too, but given that 1.24 is the minimum kubernetes version for a management cluster with 1.5, it seems this is what this statement is hinting.

With 1.5 out of support, suggesting an upgrade to 1.24 now makes less sense than before...

Reply:
Before this PR, it was saying that folks have to bump to 1.20. Which is probably because some old 1.x version required 1.20. Probably we should just drop it altogether. We don't give any recommendations of upgrade paths from v0.3 to v1.<latest> at the moment.

Reply:
New wording sounds perfect, thx!